admix/downloader.py

```diff
@@ -78,7 +79,7 @@ def download_dids(dids, num_threads=8, **kwargs):
 
 
 def download(did, chunks=None, location='.',  tries=3, metadata=True,
-             only_metadata=False, num_threads=5, rse=None):
+             only_metadata=False, num_threads=8, rse=None, stage=None):
```

Comment:
Is there a motivation for bumping the download thread number to 8?

Reply:
not really, it was probably due to many tests I have done. I will have it back to 5